### PR TITLE
Uplink: Combat Bakery Kit

### PR DESCRIPTION
**Pull request by themias**
## About the PR

This PR ports the Combat Bakery Kit from SS13 for mimes and chefs. The disintegrating box provides one baguette sword and two throwing star croissants. The main upside is that they are stealthy, looking just like their regular counterparts. They are also edible, so you can destroy the evidence after a kill!

Baguettes can now be worn on the belt, and regular ones do one blunt damage. Mimes also start with a regular baguette in their survival kits, to make them more common. I also updated the baguette sprites to look better for the swinging animation.

I haven't touched weapons or uplink items before, so I might be way off on the balance. Please leave feedback if this is too expensive/cheap or strong/weak. 

I decided against adding recipes for making more baked weapons, since secretly learning a recipe would require either adding a component to a player (bad) or would require some implant (easy to meta since it is job-locked).

## Why / Balance

I think stealth syndie items are cool, and create more possibilities for creative antag-ing. I also love the idea of a syndie mime sword fighting the captain with a baguette.

## Media

https://github.com/user-attachments/assets/3236a998-ad4b-4134-b252-5688b2af11d4

## Requirements

- [x] I have read and I am following the [Pull Request Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html). I understand that not doing so may get my pr closed at maintainer’s discretion
- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl:
- add: Added the Combat Bakery Kit to the syndicate uplink. Includes a deadly edible baguette sword and throwing star croissants for 3 TC.
- tweak: Baguettes can be worn on the belt and normal ones do one blunt damage.
- tweak: Mimes have a baguette instead of an MRE in their survival kits.

### CODE REVIEW

**Comment by beck-thompson** [comment]
This is a fun idea! In the future I'm also going to make some kind of system so after eating a cake with an explosive in it, it'll automatically go off (see #31015). When I'm done I'll add it to the kit!

**Comment by Emisse** [comment]
goncflicts

**Review by Emisse — APPROVED** [review]

**Comment by TheDoctor1977** [comment]
It should probably just be a normal box, I can easily imagine someone opening this for the first time and thinking it’s broken because half of the kit drops on the floor.

File: Resources/Prototypes/Entities/Objects/Weapons/Melee/baguette.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+- type: entity
+  name: baguette
+  parent: FoodBreadBaguette
+  id: WeaponBaguette
+  description: Bon appétit!
+  suffix: Weapon
+  components:
+  - type: MeleeWeapon
+    wideAnimationRotation: -135
```
Comment (TheShuEd) [inline Resources/Prototypes/Entities/Objects/Weapons/Melee/baguette.yml:9]: the rotation angle needs to be edited, now the animation does not quite match the angle of the sprite

File: Resources/Prototypes/Entities/Objects/Weapons/Melee/baguette.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    damage:
+      types:
+        Blunt: 16
+    soundHit:
+      path: /Audio/Weapons/bladeslice.ogg
```
Comment (Hrosts) [inline Resources/Prototypes/Entities/Objects/Weapons/Melee/baguette.yml:14]: If it deals blunt, why the slicing sound?

**Comment by ElPidor** [comment]
6 tc is kinda very veeeeeeery huge stuff.
It was good in ss13 because you could've learn receipt of making them

File: Resources/Prototypes/Entities/Objects/Weapons/Melee/baguette.yml (hunk 2 above)
Comment (themias) [reply Resources/Prototypes/Entities/Objects/Weapons/Melee/baguette.yml:14]: Good catch, I originally had it Blunt but meant to move it to slash. I must somehow dropped that from the diff. Derp.

**Comment by themias** [comment]
> 6 tc is kinda very veeeeeeery huge stuff. It was good in ss13 because you could've learn receipt of making them

That's very fair. I see the energy dagger is concealable and does 20 damage for 2 TC. I went ahead and reduced the cost of this one to 3 TC. If I can figure out a good way of doing the recipe in the future, then I'd bump it back up.

**Comment by themias** [comment]
> It should probably just be a normal box, I can easily imagine someone opening this for the first time and thinking it’s broken because half of the kit drops on the floor and they don’t notice.

I made it a disintegrating box since I was worried about it being meta'd too easily since it is job-locked to chef and mime.

Maybe in this case it could look like a plain box or a survival box? I think all other traitor items either leave no box behind or leave obvious evidence, so this would be something new.

**Comment by SlamBamActionman** [comment]
It is a bit weird that the box opens up to reveal a 3x3 storage only to be immediately deleted afterwards. I think it's fine if it's just a normal box; it can either look like a non-specific 3x3 cardboard box (we have those) or a box that is more obvious ("combat bakery box") but can be destroyed in the recycler. Having things drop on the ground probably doesn't feel too great.

**Comment by themias** [comment]
> It is a bit weird that the box opens up to reveal a 3x3 storage only to be immediately deleted afterwards. I think it's fine if it's just a normal box; it can either look like a non-specific 3x3 cardboard box (we have those) or a box that is more obvious ("combat bakery box") but can be destroyed in the recycler. Having things drop on the ground probably doesn't feel too great.

That's true, having to grab stuff off of the floor isn't great. I'll switch it to a normal box.

**Pull request merged**